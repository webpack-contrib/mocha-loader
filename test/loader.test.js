import webpack from './helpers/compiler';

describe('mocha-loader', () => {
  it('works', async () => {
    const stats = await webpack('fixture.js', {
      loader: {
        test: /fixture\.js$/,
        options: {},
      },
    });
    const statsJson = stats.toJson();

    expect(statsJson.errors).toEqual([]);
    expect(statsJson.warnings).toEqual([]);
    expect(statsJson.modules[0].source).toMatchSnapshot();
  });
});
