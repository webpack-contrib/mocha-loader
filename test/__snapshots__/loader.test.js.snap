// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mocha-loader executes mocha tests when evaluating bundle in browser: errors 1`] = `Array []`;

exports[`mocha-loader executes mocha tests when evaluating bundle in browser: warnings 1`] = `Array []`;
